### DISCUSSION TITLE
Breakpoints Question

### DISCUSSION THREAD

**Discussion by michaeltoohig** [element-plus/element-plus#10504] [Q&A] [answered]
Coming from using Vuetify last, I found the global breakpoint helpers convenient and curious if there was similar feature in element plus? For example with Vuetify I could easily write logic based on screen size with the following.

```
$vuetify.breakpoint.mobile
$vuetify.breakpoint.smAndUp
$vuetify.breakpoint.md
$vuetify.breakpoint.mdAndDown
etc.
```

Ultimately, I want to know if the screen size is small so I can set the dialogs to fullscreen and I'd ideally use the breakpoints defined by my element plus theme rather than adding some sort of watcher myself and having to keep two copies of breakpoint values in the component and in the theme.

**Comment by ryuhangyeong** [comment, marked as answer]
ep does not support attributes internally. (but, `Layout` components support conditional endpoints.)

My personal recommendation is vueuse `useBreakpoints`.

https://vueuse.org/core/usebreakpoints/#usebreakpoints
